front/ProfilePage.tsx

```diff
@@ -55,8 +55,8 @@ const ProfileHoc = (tab) => {
                     alt="User's profile image"
                     className="user-img"
                   />
-                  <h4>{username}</h4>
-                  <p>{bio}</p>
+                  <h4 data-cy="profile-username">{username}</h4>
```

Comment:
Add data-cy attribute for email and password since the task requires updating bio, username, email, and password.

Reply:
done